zerver/openapi/openapi.py

```diff
@@ -444,8 +464,11 @@ def validate_against_openapi_schema(
     if (endpoint, method) in EXCLUDE_DOCUMENTED_ENDPOINTS:  # nocoverage
         return True
     # Check if the response matches its code
-    if status_code.startswith("2") and (
-        content.get("result", "success").lower() != "success"
+    if (
+        status_code.startswith("2")
+        and response.content_type == "application/json"
+        and response.data is not None
+        and orjson.loads(response.data).get("result", "success").lower() != "success"
     ):  # nocoverage
         raise SchemaError("Response is not 200 but is validating against 200 schema")
```

Comment:
Is this exception description bogus? It's definitely 2xx, just not labeled as a success, I think?

Reply:
To be honest, I don’t understand what this is supposed to be checking for. Our OpenAPI document already provides the expected format for our 200 responses. Let’s just delete this?

Reply:
Deleted.

Reply:
Works for me.